browser/src/map/Clipboard.js

```diff
@@ -38,8 +38,8 @@ L.Clipboard = L.Class.extend({
 		this._dummyPlainDiv = null;
 		this._dummyClipboard = {};
 
-		// Tracks waiting for UNO commands to complete
-		this._commandCompletion = [];
+		// Is there an awaiting command.
+		this._commandCompletion = false;
```

Comment:
This is a fair idea (stopping this from being an array as it can only have one item), but if you want to have it as a promise then you need to save the resolve/reject somewhere ... and you do need it to be a promise or similar because otherwise you can't wait for it ... and you do need to wait for it (at least on mobile) because otherwise you run into data races that lead to data being miscopied